contracts/eosio.msig/include/eosio.msig/eosio.msig.hpp

```diff
@@ -172,5 +176,46 @@ namespace eosio {
          };
 
          typedef eosio::multi_index< "invals"_n, invalidation > invalidations;
+
+      private:
+         template<typename Function>
+         bool approvals_satisfy_trx_authorization(name proposer, name proposal_name, const std::vector<char>& packed_trx, Function&& table_op) {
+            std::vector<permission_level> approvals = get_approvals_and_adjust_table(proposer, proposal_name, table_op);
```

Comment:
I would prefer to see `approvals_satisfy_trx_authorization` not take `table_op` and to just pass in the no-op lambda for the `get_approvals_and_adjust_table` call here.

Reply:
As previously stated, I opted to just remove this function entirely.